deps/Makefile

```diff
@@ -1516,21 +1594,21 @@ get-fftw: get-fftw-single get-fftw-double
 configure-fftw: configure-fftw-single configure-fftw-double
 compile-fftw: compile-fftw-single compile-fftw-double
 check-fftw: check-fftw-single check-fftw-double
-install-fftw: check-fftw-single check-fftw-double # do these serially, to prevent write conflicts
```

Comment:
not sure. we've been installing them in separate folders since usr-staging. would `cp` / `tar x` care?